ouroboros-consensus-test/ouroboros-consensus-test.cabal

```diff
@@ -309,3 +309,36 @@ test-suite test-infra
                        -Wredundant-constraints
                        -Wmissing-export-lists
                        -fno-ignore-asserts
+
+-- test-examples is just used to type-check tutorial code:
+                          
+test-suite test-examples
+  type:                exitcode-stdio-1.0
+  hs-source-dirs:      test-examples
+  main-is:             Main.hs
+  other-modules:       Test.Tutorial
+                     , Test.Tutorial2
+                     
+  build-depends:       base
```

Comment:
We typically try to keep this list alphabetical (except the `base` dep) and generally try for one big group or maybe one "local" group and one "non-local" group, etc.

So please at least combine the first two groups and sort them.

Reply:
I have a PR open (and will be for a while) that adds a `ouroboros-consensus-docs` package.

That seems like a better fit to me---though `ouroboros-consensus-test` also seems quite reasonable.

What do you think, @dnadales ?

Reply:
> So please at least combine the first two groups and sort them.

done.

Reply:
> What do you think, @dnadales ?

`-docs` seems a better place. However, we could leave this here for the time being, and add an additional task to your PR to move `test-examples`. I'm proposing this since your PR Nick will probably take longer to merge.